LuisPalominoTrevilla/Guessit-front
# Feat/GI-15/notifications

## Prerequisites

- `npm install`

## Test

- Try to register with an invalid email, a warning notification should appear
- Try to login with invalid credentials, an error notification should appear
